### PR TITLE
PRIME-2042 Dont abbreviate BCSC

### PR DESCRIPTION
**Pull request by AMiKaH**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/sat-eforms/shared/modules/sat-eforms-login-page/sat-eforms-login-page.component.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class SatEformsLoginPageComponent implements OnInit {
     private authService: AuthService,
   ) {
     this.title = 'Enrol for access to Special Authority eForms';
-    this.loginLabel = 'Enrol using your BCSC';
+    this.loginLabel = 'Enrol using your BC Service Card';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/sat-eforms/shared/modules/sat-eforms-login-page/sat-eforms-login-page.component.ts:27]: That should be "BC Services Card" (https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card)

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/StatusReasonConfiguration.cs [hunk 2]
```diff
@@ -26,7 +26,7 @@ public override IEnumerable<StatusReason> SeedData
                     new StatusReason { Code = (int)StatusReasonType.IdentityProvider,             Name = "User authenticated with a method other than BC Services Card"                   },
                     new StatusReason { Code = (int)StatusReasonType.RequestingRemoteAccess,       Name = "User has Requested Remote Access"                                               },
                     new StatusReason { Code = (int)StatusReasonType.NoAssignedAgreement,          Name = "Terms of Access to be determined by an Adjudicator"                             },
-                    new StatusReason { Code = (int)StatusReasonType.NoVerifiedAddress,            Name = "No address from BCSC. Enrollee entered address."                                },
+                    new StatusReason { Code = (int)StatusReasonType.NoVerifiedAddress,            Name = "No address from BC Service Card. Enrollee entered address."                     },
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/StatusReasonConfiguration.cs:29]: I don't _think_ this is necessary (but is, admittedly, very thorough). We are primarily concerned with unifying the messaging to the Enrollees and not so much with the system-level messages to the adjudicators

File: prime-angular-frontend/src/app/modules/sat-eforms/shared/modules/sat-eforms-login-page/sat-eforms-login-page.component.ts (hunk 1 above)
Comment (AMiKaH) [reply prime-angular-frontend/src/app/modules/sat-eforms/shared/modules/sat-eforms-login-page/sat-eforms-login-page.component.ts:27]: I had it as **Services** but then looked at the ticket and it said **Service** 😆 , will change it back.

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/StatusReasonConfiguration.cs (hunk 2 above)
Comment (AMiKaH) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/StatusReasonConfiguration.cs:29]: I was going by the ticket where it said the entire prime app shouldn't have any abbreviation. Also confirmed it with Jason before committing that change.

**Review by Paahn — APPROVED** [review]
✌️

**Review by johnnyl6 — APPROVED** [review]
LGTM

File: prime-dotnet-webapi/Migrations/20211217163513_ChangedBCSCReference.cs [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
...) cease all access or use of\r\n      PharmaNet.\r\n    </p>\r\n\r\n    <p>\r\n      Any written notice to you under (i) above will be in writing and delivered by the Province to you using any of the\r\n      contact mechanisms identified by you in PRIME, including by mail to a specified postal address, email to a\r\n      specified email address or text message to a specified cell phone number. You may be required to click a URL link\r\n      or log into PRIME to receive the contents of any such notice.\r\n    </p>\r\n\r\n  </li>\r\n  {$lcPlaceholder}\r\n  <li>\r\n\r\n    <p class=\"bold underline\">\r\n      Governing Law\r\n    </p>\r\n\r\n    <p>\r\n      These terms will be governed by and will be construed and interpreted in accordance with the laws of British\r\n      Columbia and the laws of Canada applicable therein.\r\n    </p>\r\n\r\n    <p>\r\n      Unless otherwise specified, a reference to a statute or regulation by name means the statute or regulation of\r\n      British Columbia of that name, as amended or replaced from time to time, and includes any enactment made under the\r\n      authority of that statute or regulation.\r\n    </p>\r\n\r\n  </li>\r\n</ol>\r\n");
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Migrations/20211217163513_ChangedBCSCReference.cs]: could you re-generate this migration? you seem to have some issues with line endings here

Comment (AMiKaH) [reply prime-dotnet-webapi/Migrations/20211217163513_ChangedBCSCReference.cs]: Hmmm, it's odd that this is here. This is what the file was supposed to look like
```
using Microsoft.EntityFrameworkCore.Migrations;

namespace Prime.Migrations
{
    public partial class ChangedBCSCReference : Migration
    {
        protected override void Up(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.UpdateData(
                table: "StatusReasonLookup",
                keyColumn: "Code",
                keyValue: 17,
                column: "Name",
                value: "No address from BC Service Card. Enrollee entered address.");
        }

        protected override void Down(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.UpdateData(
                table: "StatusReasonLookup",
                keyColumn: "Code",
                keyValue: 17,
                column: "Name",
                value: "No address from BCSC. Enrollee entered address.");
        }
    }
}
```

Comment (AMiKaH) [reply prime-dotnet-webapi/Migrations/20211217163513_ChangedBCSCReference.cs]: Not odd, but I don't understand why/how that change happened.

**Review by james-hollinger — APPROVED** [review]
LGTM

**Pull request merged**